desktop/src/dashboard/report-widget/report-widget.component.scss

```
@@ -0,0 +1,39 @@
+.report-widget {
+  &__stage {
+    position: relative;
+    // The report renders its full dataset; cap the visible area and scroll so a long
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)